dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/AbstractClouderaManagerTask.java

```diff
@@ -65,6 +65,10 @@ protected <T> T parseJsonStringToObject(String jsonString, Class<T> type)
     return objectMapper.readValue(jsonString, type);
   }
 
+  protected <T> T parseJsonStreamToObject(InputStream src, Class<T> type) throws IOException {
+    return objectMapper.readValue(src, type);
+  }
+
   protected String parseObjectToJsonString(Object obj) throws JsonProcessingException {
```

Comment:
It's part of a different PR, but object to string it's not `parse`, it serialization. I would like to suggest to reuse the names from    `objectMapper`.

Reply:
I agree about `parsing` vs `serialization`. But some names in `objectMapper` are not obvious. For example the method `.writValueAsString(obj)`. This method only creates a String and doesn't write anything into the output stream.